apps/web/src/index.d.ts

```diff
@@ -2,3 +2,5 @@
 declare module "*.jpg";
 declare module "*.png";
 declare module "*.svg";
+declare module "*.pdf";
```

Comment:
Maybe update comment since these are not images.
Also, missing *.doc.

Reply:
Oops, updated in [update static asset module declarations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/3649561bd8518ba6e5b66afa60c23b161e4b0bbd)